core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -485,4 +485,13 @@ package object config {
         "array in the sorter.")
       .intConf
       .createWithDefault(Integer.MAX_VALUE)
+
+  private[spark] val SHUFFLE_MAP_OUTPUT_STATISTICS_PARALLEL_AGGREGATION_THRESHOLD =
```

Comment:
`spark.adaptive.map.statistics.cores` should also be a config entry like this

Reply:
`spark.sql.adaptive.xxx` already exists, will this be a problem?

Reply:
Really? I grep the code base but can't find it.

Reply:
Like https://github.com/gczsjdy/spark/blob/11b60af737a04d931356aa74ebf3c6cf4a6b08d6/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L204-L204

Reply:
I think that's not a big problem, adaptive execution need both core and sql code, so both confs are needed.

Reply:
I don't get it. You showed me that `spark.sql.adaptive.xxx` have config entries, why `spark.adaptive.map.statistics.cores` doesn't need config entry?

Reply:
`spark.adaptive.map.statistics.cores` needs config entry, but I thought adaptive.xxx item has been put under `spark.sql.` already, so it might be inconsitent. Now I think it's no big deal.

Reply:
There is also a `spark.shuffle.mapOutput.dispatcher.numThreads` in this file without config entry, do I need to add one?

Reply:
yea let's add it. BTW shall we also use `mapOutput` instead of `mapOutputStatistics`?

Reply:
Actually there are 3 confs like that... all need?